### DISCUSSION TITLE
Welcome to typedoc-theme-yaf Discussions!

### DISCUSSION THREAD

**Discussion by citkane** [citkane/typedoc-theme-yaf#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by citkane** [comment]
Hi @toebeann, 

I hope you had a good Chrismas and that 2023 has started well for you. :smiley:

[This](https://github.com/citkane/typedoc-theme-yaf) is the TypeDoc theme I started building last year.  
It is almost ready for an Alpha release to NPM, but your initial thoughts and opinions would be valued before I do that.

You can have a look at the [live documentation](https://citkane.github.io/typedoc-theme-yaf/) to get a feel for it. There are still a few mobile rendering and more issues...

All the best,
@citkane

**Reply by toebeann** [reply]
Hi! 2023 has indeed had a good start for myself, hope the same is true for you

This theme overall looks really quite nice, I'm a fan! I look forward to seeing it completed 😄

**Reply by citkane** [reply]
Thanks @toebeann,

All good thanks! Another year, flying by...

I will get release 0.0.0 on NPM when I have `typedoc-plugin-versions` implemented and then some form of search.
Glad you like it.
Spread the word!

:hear_no_evil: :see_no_evil: :speak_no_evil: :mega:

**Reply by citkane** [reply]
That should be good for a kick at the ole proverbial tyres now :)

https://www.npmjs.com/package/typedoc-theme-yaf?activeTab=readme

```
npm i  -D typedoc-theme-yaf
```

```
{
    "plugin": ["typedoc-theme-yaf"],
    "theme": "yaf"
}
```

**Reply by citkane** [reply]
Remember to get rid of `typedoc-plugin-versions` in your options, it is baked into the new theme.

**Reply by toebeann** [reply]
Finally got a chance to more thoroughly play with this today! Overall I really like it, though there are definitely some issues at the moment which are stopping me from using it in production. My biggest issue is that [inline `@link` tags do not resolve correctly when combined with `@params` tags and some `@see` tags](https://github.com/citkane/typedoc-theme-yaf/issues/11). I tend to use `@link` tags a lot to make navigating to relevant parts of the documentation a breeze!

**Reply by citkane** [reply]
On it @toebeann ,
It's been a very busy time out in the physical world, and my screen time has been horribly slashed!
All good :)
